### PR TITLE
ENYO-1224: updated the contextual popup sample to check modal property

### PR DESCRIPTION
**Pull request by anishlg**
Changed the contextual popup sample file to include the modal checkbox button to verify the modal property

DCO-1.1-Signed-Off-By: Anish TD anish.td@lge.com

### CODE REVIEW

**Comment by KunmyonChoi** [comment]
Hii @anishlg , thanks for the PR. It works well. 
I just made minor changes on sample to sync up state of each properties.

File: src/moonstone-samples/src/ContextualPopupSample.js [hunk 1]
```diff
@@ -233,8 +231,14 @@ module.exports = kind({
 		{from: '.$.nestedRadioGroup.active.content', to: '.$.nestedRadioValue.content', transform: function(val){
 			this.dismissRadioSelection();
 			return val;
-		}}
+		}},
+		{from: '$.btnSpotlightModal.value', to: '$.buttonPopup.spotlightModal', oneWay: false},
+		{from: '$.btnSpotlightModal.value', to: '$.buttonPopup.autoDismiss', kind: InvertBooleanBinding, oneWay: false},
+		{from: '$.btnModal.value', to: '$.buttonPopup.modal', oneWay: false}
 	],
+	modelToggled: function(inSender, inEvent) {
```
Comment (aarontam) [inline src/moonstone-samples/src/ContextualPopupSample.js]: This handler is no longer needed, right? It should also have probably been `modalToggled`, originally. :smile:

Comment (KunmyonChoi) [inline src/moonstone-samples/src/ContextualPopupSample.js]: Right!

**Comment by webOS101** [comment]
Love the use of InvertBooleanBinding but I find this to be confusing to have two buttons that are locked to each other.

**Pull request closed without merging**